multiuser/keycloak/che-multiuser-keycloak-server/pom.xml

```diff
@@ -147,6 +147,11 @@
             <artifactId>javax.servlet-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
```

Comment:
is it lib already defined in https://github.com/eclipse/che-parent/blob/master/pom.xml ? CQ approved etc?

Reply:
deps moved from che-parent https://github.com/eclipse/che/blob/master/pom.xml. This build and test-lib already has a cq in Eclipse db (https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20192) and according to the latest simplification of process, we don't need to create a PB cq for it.